includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -133,13 +133,16 @@ protected function should_show_facebook() {
 	}
 
 	/**
-	 * Is the WooCommerce Admin actively included in the WooCommerce core?
-	 * Based on presence of a basic WC Admin function.
+	 * Is the WooCommerce Admin Plugin installed already?
 	 *
 	 * @return boolean
 	 */
-	protected function is_wc_admin_active() {
-		return function_exists( 'wc_admin_url' );
+	protected function is_wc_admin_plugin_active() {
+		$is_plugin_active = false;
+		if ( class_exists( 'Automattic\WooCommerce\Admin\Compose\Package' ) ) {
+			$is_plugin_active = Automattic\WooCommerce\Admin\Compose\Package::is_package_active();
```

Comment:
This seems off. If `is_package_active()` returns true, that would mean the plugin is **in**active.

Reply:
Why not use [`is_plugin_active()`](https://developer.wordpress.org/reference/functions/is_plugin_active/) here?